src/recipes/chickoo_lassi.json

```
@@ -0,0 +1,47 @@
+{
+    "name": "<Chickoo lassi>",
+    "description": "<A chilled glass of lassi is just the perfect way to refresh during the summer season.
+	                 Here's an easy and quick recipe to prepare delectable lassi at home with the flavours of chickoo fruit
+	                 blended with curd, milk,cardamom and honey>",
+    "github": "<Priyanka788>",
```

Remove all braces from all fields.
```suggestion
    "github": "Priyanka788",
```